From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: axjp <63340001+Alpha-404@users.noreply.github.com>
Date: Sun, 16 Nov 2025 12:33:06 +0000
Subject: [PATCH] 8633: Fix typo in Languages.htm regarding language change

Corrected a typo in the instructions for changing the processor language.
---
 .../main/help/help/topics/LanguageProviderPlugin/Languages.htm  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Ghidra/Features/Base/src/main/help/help/topics/LanguageProviderPlugin/Languages.htm b/Ghidra/Features/Base/src/main/help/help/topics/LanguageProviderPlugin/Languages.htm
index c5d5df96e7..a266d9cba8 100644
--- a/Ghidra/Features/Base/src/main/help/help/topics/LanguageProviderPlugin/Languages.htm
+++ b/Ghidra/Features/Base/src/main/help/help/topics/LanguageProviderPlugin/Languages.htm
@@ -52,7 +52,7 @@ In addition, the program file may not be open in a tool when changing
 the assigned processor language.</p>
 <p> To change the processor language, right-click on the prorgam file
 within the <span style="font-style: italic;">Project Window</span>
-data tree, and setlect <span style="font-weight: bold;">Set Language...</span>
+data tree, and select <span style="font-weight: bold;">Set Language...</span>
 from the popup menu.&nbsp; Since setting the language is such a major
 change, the following warning will appear.</p>
 <p align="center"><img alt="" src="images/Warning.png"></p>
-- 
2.45.1

